frequency-chain/testnet-faucet
# Delete request lib because it's not being used

The faucet is actually not even using the deprecated `request` package.  So it's removed from package.json.  I tested only with a localhost against a local chain, because the request package was originally used only by the faucet server.

Closes #24 

## To verify
1. first launch a local frequency chain.
2. launch  the server
```shell
git fetch -ftp
git co feat/replace-request
yarn install --frozen-lockfile && yarn dev
```
3. launch the client in another terminal
```shell
cd client
yarn install --frozen-lockfile && yarn dev -- --open
```
4.  Fill in the faucet form with a valid address and submit.
5. Verify that the address receives 10 UNIT
